guide/src/config.md

```diff
@@ -16,163 +16,16 @@ name of the command to run, eg. `hermes -c my_config.toml query connection chann
 hermes [-c CONFIG_FILE] COMMAND
 ```
 
-## Sections
```

Comment:
Can we please leave this file untouched for the moment?

There are two reasons for delaying changes to this files:
- there may still be v0.5.0 users which do not have the new config.toml with inline documentation
- but more importantly: it seems the DeX team had a misconfiguration of `trust_threshold` in config.toml and it may be better to try and simplify this file (e.g., by removing sensitive options that have clear defaults) while documenting _all_ possible options in some place (here).


I will open another issue to to follow-up on the second bullet above.